web/app/services/session.js

```diff
@@ -33,10 +33,23 @@ export default Ember.Service.extend({
     })
   },
   logout(){
-    this.set("currentUser", null)
-    Cookies.remove('userId')
+    this.set("currentUser", null);
   },
   setLoggedInUser(username) {
     this.set("currentUser", username);
+    //saving username to localStorage for persistence even after browser refresh
+    localStorage.setItem('currentUser', username);
+  },
+  getActiveUser(){
+    let activeUser = this.get('currentUser');
+    if (activeUser) {
+      return activeUser;
+    } else {
+      activeUser = localStorage.getItem('currentUser');
```

Comment:
Additional check that would be good to have (You may like to store these values in localStorage object as well): 
- timeout based mechanism for logging out users. (You can set time out as the one already being used for java session time out.

Reply:
The expiration of the user session is been handled by the `session.id` cookie, so as the user's session gets expired then the `session.id` cookie will be removed and the user will prompted to login again and at that time `currentUser` in the localStorage will be update to the current active user.